client/app/components/app-header/AppHeader.jsx

```diff
@@ -0,0 +1,252 @@
+/* eslint-disable no-template-curly-in-string */
+
+import React, { useRef } from 'react';
+import { react2angular } from 'react2angular';
+
+import Dropdown from 'antd/lib/dropdown';
+import Button from 'antd/lib/button';
+import Icon from 'antd/lib/icon';
+import Menu from 'antd/lib/menu';
+import Input from 'antd/lib/input';
+import Tooltip from 'antd/lib/tooltip';
+
+import DropdownFavorites from './components/FavoritesDropdown';
+import { HelpTrigger } from '@/components/HelpTrigger';
+import CreateDashboardDialog from '@/components/dashboards/CreateDashboardDialog';
+
+import { currentUser, Auth, clientConfig } from '@/services/auth';
+import { $location, $route } from '@/services/ng';
+import navigateTo from '@/services/navigateTo';
+import { Dashboard } from '@/services/dashboard';
+import { Query } from '@/services/query';
+import frontendVersion from '@/version.json';
+import logoUrl from '@/assets/images/redash_icon_small.png';
+
+import './AppHeader.less';
+
+function onSearch(q) {
+  $location.path('/queries').search({ q });
+  $route.reload();
+}
+
+function Desktop() {
+  return (
+    <div className="app-header" data-platform="desktop">
+      <div>
+        <Menu mode="horizontal" selectable={false}>
+          <Menu.Item key="dashboards" className="dropdown-menu-item">
+            <Button onClick={() => navigateTo('dashboards')}>
+              Dashboards
+            </Button>
+            <DropdownFavorites fetch={Dashboard.favorites} urlTemplate="dashboard/${slug}" />
+          </Menu.Item>
+          <Menu.Item key="queries" className="dropdown-menu-item">
+            <Button onClick={() => navigateTo('queries')}>
+              Queries
+            </Button>
+            <DropdownFavorites fetch={Query.favorites} urlTemplate="queries/${id}" />
+          </Menu.Item>
+          <Menu.Item key="alerts">
+            <Button onClick={() => navigateTo('alerts')}>
+              Alerts
+            </Button>
+          </Menu.Item>
+        </Menu>
+        <Dropdown
+          trigger={['click']}
+          overlay={(
+            <Menu>
+              {currentUser.hasPermission('create_query') && (
+                <Menu.Item key="new-query">
+                  <a href="queries/new">New Query</a>
+                </Menu.Item>
+              )}
+              {currentUser.hasPermission('create_dashboard') && (
+                <Menu.Item key="new-dashboard">
+                  <a className="clickable" onClick={CreateDashboardDialog.showModal}>New Dashboard</a>
+                </Menu.Item>
+              )}
+              <Menu.Item key="new-alert">
+                <a href="alerts/new">New Alert</a>
+              </Menu.Item>
+            </Menu>
+          )}
+        >
+          <Button type="primary" data-test="CreateButton">
+            Create <Icon type="down" />
+          </Button>
+        </Dropdown>
+      </div>
+      <div className="header-logo">
+        <a href={clientConfig.basePath}>
```

Comment:
```suggestion
        <a href="./">
```

It annoys me that this depends on a backend config and it doesn't fit our Redash-preview case, for example... I wonder if the above works for Multi-org :thinking:

Reply:
That actually works for multi-org!
Learned sth new today :)